inc/namespace.php

```diff
@@ -1479,6 +1534,7 @@ function get_autosuggest_fields( ?string $type = null ) : array {
 	// Add default autosuggest fields.
 	if ( isset( $fields['post'] ) ) {
 		$fields['post'] = [ 'post_title' ];
+		$fields['post'] = [ 'post_name' ];
```

Comment:
Not sure I understand this change -- we're overwriting the value here, should we be setting this to a list of `post_title` _and_ `post_name`? What's the intent of this line?

Reply:
Oh whoops, that was some other work in progress unrelated to this PR, I'll remove it.